core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -1389,6 +1389,11 @@ public function addCtableTags($strTable, $intId, &$tags)
 		}
 	}
 
+	protected function getFormFieldSuffix()
```

Comment:
Should this function get a return type hint? Or at least a phpdoc comment? Otherwise you again wouldn't know if you're dealing with an integer or a string.

Reply:
Added in eb52224ac0a85237dc9e1b936cb47496ddc2387c.